src/libs/actions/IOU.ts

```diff
@@ -421,9 +421,12 @@ function setCustomUnitRateID(transactionID: string, customUnitRateID: string) {
 }
 
 /** Update transaction distance rate */
-function updateDistanceRequestRate(transactionID: string, rateID: string, policyID: string) {
+function updateDistanceRequestRate(transactionID: string, rateID: string, policyID: string, taxAmount?: number) {
     Onyx.merge(ONYXKEYS.NVP_LAST_SELECTED_DISTANCE_RATES, {[policyID]: rateID});
     Onyx.merge(`${ONYXKEYS.COLLECTION.TRANSACTION_DRAFT}${transactionID}`, {comment: {customUnit: {customUnitRateID: rateID}}});
+    if (taxAmount) {
+        Onyx.merge(`${ONYXKEYS.COLLECTION.TRANSACTION_DRAFT}${transactionID}`, {taxAmount});
+    }
```

Comment:
I think we should create separate function to update taxAmount and call it separately